docs/reference/commands.md

```diff
@@ -236,7 +238,7 @@ List all extensions.
 - `--columns COLUMNS`
     - Comma-separated list of columns to show. Available columns: name. Defaults to `--columns name`.
 - `-s`, `--simple`
-    - Only show table data, without headers. Use this option for generating machine-readable output.
+    - Only show table data, without headers. Use this option for generating machine- readable output.
```

Comment:
Nit: This isn't correct

Reply:
Btw these files in `docs/reference/..` are autogenerated, and are overwritten anyway when the actual documentation is built.

There's just a weird bug that if these autogenerated files are TOTALLY omitted, then the page does NOT get generated (specifically, our script generates the file, but mkdocs doesn't pick it up). So there needs to be *something* in the source control.

But the end result is still formatted correctly:

<img width="726" alt="Screenshot 2025-05-09 at 09 14 59" src="https://github.com/user-attachments/assets/17fc8f59-84cc-4582-90c5-c37b52b237fa" />